docs/DeveloperGuide.md

```diff
@@ -891,262 +891,264 @@ testers are expected to do more *exploratory* testing. Alternative test cases wi
+2. No person in the list has any scheduled meeting.
 
-**Example test case:** `reschedule 1 s/01-12-2024T09:30 e/01-12-2024T12:30`.
+**Example Test Case**: `schedule 1 s/12-12-2024T16:00 e/12-12-2024T17:00 mr/Online Meeting`.
 
 **Expected Result:**
-1. The meeting details of the specified person is updated with the provided rescheduled meeting date and time.
-2. *Command Result Box* outputs a successful execution message with the rescheduled meeting information.
+1. The meeting details of the specified person is updated with the provided meeting date and time.
+2. *Person List* updates the list of contact with the person with details of the meeting.
+3. *Command Result Box* outputs a successful execution message with the details of the meeting with the specified person.
 4. *Command Box* is cleared.
 
 **Alternative Test Cases:**
-- `reschedule 2 s/20-06-2024T09:30 e/20-06-2024T12:30`.
-- `reschedule 3 s/15-09-2024T10:00 e/15-09-2024T11:00`.
+- `schedule 2 s/12-12-2024T16:00 e/12-12-2024T17:00`.
+- `schedule 3 s/12-12-2024T16:00 e/12-12-2024T17:00 mr/Meeting @ House`.
 
-#### Rescheduling a meeting today to another day successfully 
+#### Scheduling a meeting with a person which starts today successfully
 
 **Prerequisites:**
 1. List all persons using the `list` command. At least 5 persons are in the list.
-2. The first person in the list has a scheduled meeting today.
+2. No person in the list has any scheduled meeting.
+3. **The provided meeting start date must be the same as the system's current date**
 
-**Example test case:** `reschedule 1 s/01-12-2024T09:30 e/01-12-2024T12:30`.
+**Example test case:** `schedule 1 s/12-12-2024T12:00 e/12-12-2024T14:00`.
 
 **Expected Result:**
-1. The meeting details of the specified person is updated with the provided rescheduled meeting date and time.
-2. *Today’s Meeting List* shows the list of today’s meetings without the person with the rescheduled meeting.
-3. *Command Result Box* outputs a successful execution message with the rescheduled meeting information.
-4. *Command Box* is cleared.
+1. The meeting details of the specified person is updated with the provided meeting date and time.
+2. *Person List* updates the list of contact with the person with details of the meeting.
+3. *Today’s Meeting List* shows the list of today’s meetings with the specified person.
+4. *Command Result Box* outputs a successful execution message with the details of the meeting with the specified person.
+5. *Command Box* is cleared.
 
-#### Invalid value or command format used for rescheduling a meeting
+#### Scheduling a meeting with a person who has a scheduled meeting
 
 **Prerequisites:**
-1. List all persons using the `list` command. At least 1 person is in the list.
-2. First person in the displayed list has a scheduled meeting.
+1. List all persons using the `list` command. At least 5 persons are in the list.
+3. All persons in the list has a scheduled meeting.
```

Comment:
Just a nit but I think we should use proper ordering although jekyll will render the numbering correctly for us?